queries/user.ts

```diff
@@ -1,5 +1,25 @@
 import { User } from '@prisma/client';
 import prisma from '@/lib/prisma';
+import { getServerSession } from 'next-auth/next';
+import { authOptions } from '@/app/api/auth/options';
+import checkUserMinistryRole from '@/components/utils/checkUserMinistryRole';
+
+export async function userInfo() {
+  const session = await getServerSession(authOptions);
+  const { email: authEmail, roles: authRoles } = session
+    ? session.user
+    : {
```

Comment:
I believe we should return `null` if session does not exist and the `query` should return `null` as well if no session.
Because when `email` is undefined, it will consider the user as the admin user in the query generation.